client/src/components/ReviewsControls.jsx

```diff
@@ -1,12 +1,270 @@
+                      // onChange={this.handleInputSepNov}
+                    />
+                    Sep-Nov
+                  </label>
+                </li>
+                <li className="checkbox">
+                  <label>
+                    <input
+                      name="dec-feb"
+                      type="checkbox"
+                      //value=
+                      // checked={this.state.decFeb}
+                      // onChange={this.handleInputDecFeb}
+                    />
+                    Dec-Feb
+                  </label>
+                </li>
+              </ul>
+            </div>
+            {/* <--------------------------traveler-type--------------------> */}
+            <div className="traveler-type">
+              <div className="filter-header">Traveler type</div>
+              <ul className="filter-table">
+                <li className="checkbox">
+                  <label>
+                    <input
+                      name="families"
+                      type="checkbox"
+                      //value=
+                      // checked={this.state.families}
+                      // onChange={this.handleInputFamilies}
+                    />
+                    Families
+                  </label>
+                </li>
+                <li className="checkbox">
+                  <label>
+                    <input
+                      name="couples"
+                      type="checkbox"
+                      //value=
+                      // checked={this.state.families}
+                      // onChange={this.handleInputFamilies}
+                    />
+                    Couples
+                  </label>
+                </li>
+                <li className="checkbox">
+                  <label>
+                    <input
+                      name="solo"
+                      type="checkbox"
+                      //value=
+                      // checked={this.state.solo}
+                      // onChange={this.handleInputSolo}
+                    />
+                    Solo
+                  </label>
+                </li>
+                <li className="checkbox">
+                  <label>
+                    <input
+                      name="business"
+                      type="checkbox"
+                      //value=
+                      // checked={this.state.business}
+                      // onChange={this.handleInputbBsiness}
+                    />
+                    Business
+                  </label>
+                </li>
+                <li className="checkbox">
+                  <label>
+                    <input
+                      name="friends"
+                      type="checkbox"
+                      //value=
+                      // checked={this.state.friends}
+                      // onChange={this.handleInputFriends}
+                    />
+                    Friends
+                  </label>
```

Comment:
Hey Polina I'm thinking that you might be able to split some of your <li>'s into their own components.  For example it seems like you can probably split this logic out into its own component and pass things like "name" and the text to the child component. Then you can tailor your onClick and onChange methods to just be one function that you pass one argument into.